### PR TITLE
Client side work + added simple tests

### PR DESCRIPTION
**Pull request by Atr**
In this PR you will find that I:
- Built a barebones design for the components of the 'average stars' submodule of my App
- Implemented the ability for the main app to issue a get request to the server
- Used the get request response to populate the App's state
- Created some very simple rendering tests using React + Enzyme

### CODE REVIEW

File: client/src/app.jsx [hunk 1]
```diff
@@ -1,20 +1,78 @@
-import React, { Component } from 'react';
-import './app.css';
+import React from 'react';
+import axios from 'axios';
```
Comment (justinmeskan) [inline client/src/app.jsx:2]: I like that you used Axios instead of Jquery.

File: client/src/app.jsx [hunk 2]
```diff
@@ -1,20 +1,78 @@
...
+
+  getListingReviews(id) {
+    axios.get(`/api/listings/${id}/reviews`)
+      .then((response) => {
+        // console.log(response.data);
+        this.setState({
+          reviews: response.data,
+        });
+      })
+      .catch((error) => {
```
Comment (justinmeskan) [inline client/src/app.jsx:53]: Good job properly handling errors. It's common to forget.

File: client/src/components/ContainerReviews.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const ContainerReviews = props => (
+  <div>
+    <h3> Reviews Container </h3>
```
Comment (justinmeskan) [inline client/src/components/ContainerReviews.jsx:5]: One minor nit picky thing is indention. Here it looks like you used two spaces where in client/src/components/ContainerAverageStars.jsx line 7 and 8 it appears to be a tab equal to 4 spaces.

File: server/server.js [hunk 4]
```diff
@@ -1,13 +1,10 @@
 const express = require('express');
 const path = require('path');
-const bodyParser = require('body-parser');
```
Comment (justinmeskan) [inline server/server.js:3]: Why did you remove body-parser? I'm just curious.

File: client/src/app.jsx (hunk 1 above)
Comment (Atr) [reply client/src/app.jsx:2]: Thanks Justin!

File: client/src/app.jsx (hunk 2 above)
Comment (Atr) [reply client/src/app.jsx:53]: Thanks Justin.  Haven't quite handled them "properly" yet, but that's the goal!

File: client/src/components/ContainerReviews.jsx (hunk 3 above)
Comment (Atr) [reply client/src/components/ContainerReviews.jsx:5]: Thanks for pointing that out, I'll take a look!  I'm trying to do what the linter suggests, sometimes it's a bit particular.

File: server/server.js (hunk 4 above)
Comment (Atr) [reply server/server.js:3]: I realize that because my client doesn't need to make any post requests, I don't need it just yet!

File: client/src/components/ContainerReviews.jsx (hunk 3 above)
Comment (Atr) [reply client/src/components/ContainerReviews.jsx:5]: Dude I had tabs!  Now fixed.  Thanks for catching man, definitely not nit picky, I appreciate you pointing it out!
